python/ray/data/_internal/iterator/stream_split_iterator.py

```diff
@@ -95,7 +100,15 @@ def gen_blocks() -> Iterator[RefBundle]:
                         schema=block_ref_and_md.schema,
                     )
 
-        return gen_blocks(), self._iter_stats, False
+        # For streaming split, prefetch reporting is handled via the coordinator actor
+        # since the operator is on a different node. Create a callback that calls
+        # the coordinator actor's update_prefetch_count method with the split index.
+        def prefetch_count_update(blocks: int, bytes: int) -> None:
+            self._coord_actor.update_prefetch_count.remote(
+                self._output_split_idx, blocks, bytes
+            )
```

Comment:
we can report the prefetched size via `SpilitCoordinator.get` to avoid this extra actor call.